src/protocol.h

```diff
@@ -315,20 +312,25 @@ class CAddress : public CService
     unsigned int nTime;
 };
 
-/** getdata message types */
+/** getdata message type flags */
 const uint32_t MSG_WITNESS_FLAG = 1 << 30;
 const uint32_t MSG_TYPE_MASK    = 0xffffffff >> 2;
+
+/** getdata / inv message types.
+ * These numbers are defined by the protocol. When adding a new value, be sure
+ * to mention it in the respective BIP.
+ */
 enum GetDataMsg
 {
     UNDEFINED = 0,
-    MSG_TX,
-    MSG_BLOCK,
+    MSG_TX = 1,
+    MSG_BLOCK = 2,
     MSG_TYPE_MAX = MSG_BLOCK,
```

Comment:
MSG_TYPE_MAX is not used at all in the sources and looks strange in the middle...